kernelci.org/content/docs/team/tsc.md

```diff
@@ -0,0 +1,267 @@
+
+* [Kevin Hilman](mailto:<khilman@baylibre.com>) - `khilman`
+* [Mark Brown](mailto:<broonie@kernel.org>) - `broonie`
+* [Guillaume Tucker](mailto:<guillaume.tucker@collabora.com>) - `gtucker`
+* [Corentin Labbe](mailto:<clabbe@baylibre.com>) - `montjoie`
+* [Nikolai Kondrashov](mailto:<Nikolai.Kondrashov@redhat.com>) - `spbnick`
+* [Michał Gałka](mailto:<michal.galka@collabora.com>) - `mgalka`
+* [Alexandra Pereira](mailto:<alexandra.pereira@collabora.com>)
+* [Lakshmipathi Ganapathi](mailto:<lakshmipathi.ganapathi@collabora.com>)
+
+For general discussions, the regular
+[`kernelci@groups.io`](mailto:<kernelci@groups.io>) mailing list can be used.
+To contact only the TSC members directly, the
+[`kernelci-tsc@groups.io`](mailto:<kernelci-tsc@groups.io>) private list may be
+used instead.
+
+There are essentially three kinds of maintainer roles:
+
+[software maintainers](#software-maintainers)
+: in charge of the KernelCI software components
+
+[service maintainers](#service-maintainers)
+: in charge of the KernelCI hosted services
+
+[feature maintainers](#feature-maintainers)
+: in charge of features that span across the whole KernelCI stack
+
+[channel maintainers](#channel-maintainers)
+: in charge of the communication channels used by KernelCI
+
+
+## Software Maintainers
+
+At least one maintainer is assigned to each software component.  The work
+involves reviewing pull requests, triaging and reviving old ones, updating
+GitHub settings and facilitating the development workflow.  Having “deputy”
+maintainers whenever possible also helps with the continuity of the project.
+
+### Core tools
+
+The [core tools](/docs/core) provide the command line utilities and the
+`kernelci` Python package used to implement a KerneLCI pipeline.
+
+* Repository: [`kernelci-core`](https://github.com/kernelci/kernelci-core)
+* Maintainer: `gtucker`
+* Deputy: `mgalka`
+
+### Backend
+
+The KernelCI backend provides a [web API](https://api.kernelci.org/) to send
+build and test results and let the frontend dashboard retrieve.  It also
+generates email reports, tracks regressions and triggers automated bisections.
+
+* Main repository: [`kernelci-backend`](https://github.com/kernelci/kernelci-backend)
+* Ansible config repository: [`kernelci-backend-config`](https://github.com/kernelci/kernelci-backend-config)
+* Maintainer: `mgalka`
+* Deputy: `gtucker`
+
+### Frontend
+
+The KernelCI frontend provides a dynamic [web
+dashboard](https://kernelci.org/job/) showing the data available from the
+backend.
+
+* Main repository: [`kernelci-frontend`](https://github.com/kernelci/kernelci-frontend)
+* Ansible config repository: [`kernelci-frontend-config`](https://github.com/kernelci/kernelci-frontend-config)
+* Maintainer: Alexandra
+* Deputy: `gtucker`
+
+### KCIDB
+
+KCIDB provices a set of tools to submit kernel test data to a common database.
+
+* Main repository: [`kcidb`](https://github.com/kernelci/kcidb)
```

Comment:
Can we add [`kcidb-io`](https://github.com/kernelci/kcidb-io/) here for completeness?

Reply:
Done.